tools/cli/go-whisk/whisk/client.go

```diff
@@ -204,6 +210,8 @@ func (c *Client) NewRequest(method, urlStr string, body interface{}, includeName
         return nil, werr
     }
 
+    req.Header.Add("User-Agent", c.Config.UserAgent + "/" + c.Config.CLIVersion)
```

Comment:
Probably better to move CLIVersion logic into go-whisk-cli and just set the agent to `c.Config.UserAgent`.